fpga_interchange/chip_info.py

```diff
@@ -1080,6 +1174,7 @@ def append_bba(self, bba, label_prefix):
             'GlobalCellPOD',
             'MacroPOD',
             'MacroExpansionPOD',
+            'ClusterPOD',
```

Comment:
as you are changing the bba structures, you need to bump https://github.com/SymbiFlow/python-fpga-interchange/blob/6f8c64327afdf7b88643bad52e3b5e1caa633006/fpga_interchange/chip_info.py#L1138 and the corresponding value in nextpnr: https://github.com/YosysHQ/nextpnr/blob/13c037cc0847d8eea94dabf719ca17dad69254c9/fpga_interchange/chipdb.h#L37

Reply:
Sure, done